sample-workflows/po-lint.yml

```
@@ -0,0 +1,28 @@
+name: Linting Workflow
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  push:
+    branches:
+      - '*'
+  workflow_dispatch:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [ 3.13 ]
```

python-newest now reflects the 3.14 release.

```suggestion
        version: [ '3.14' ]
```